app/src/main/java/com/berniesanders/connect/data/ActionAlertAdapter.java

```diff
@@ -0,0 +1,73 @@
+package com.berniesanders.connect.data;
+
+import android.app.Notification;
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.TextView;
+
+import com.berniesanders.connect.R;
+import com.berniesanders.connect.view.ActionAlertViewHolder;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+/**
+ * A simple array adapter for Action Alerts
+ *
+ * Created by John on 2/21/16.
+ */
+public class ActionAlertAdapter extends RecyclerView.Adapter<ActionAlertViewHolder> {
+    
+    private final List<ActionAlert> alerts;
+    private final Callback callback;
```

Comment:
this is an example where holdr really works well...this is just a click interface, which is required since recyclerview doesn't support setting an onItemClickListener.